### ISSUE TITLE
LeafEdit - Player Editor

### ISSUE THREAD

**Issue by SuperSaiyajinStackZ** [Universal-Team/LeafEdit#23]
TODO's for the Player Editor:

**New Leaf**

- [ ] Edit Emotions.
- [ ] Edit Encyclopedia.
- [ ] Edit Gender.
- [ ] Edit Patterns. (Will be probably inside another Issue too called `LeafEdit - Pattern`.)
- [ ] Edit TPC Text.
- [ ] Inject TPC Image.
- [ ] Edit House?

**Wild World**
- [ ] Edit Face.
- [ ] Edit Name.
- [ ] Edit Gender.
- [ ] Edit Emotions. (IIRC there are 4 Slots for Emotions.)
- [ ] Edit Bank Amount.

If I forgot something on the Player Editor, please comment under this Issue, thank you.

**Comment by SuperSaiyajinStackZ** [comment]
A weird Issue is currently going on by Wild World saves.

- Gender doesn't really Change on the EUR save. (Changed only once by a few tries.)
- Somehow Player Name changes affects this too(?).

[ACWW-Saves.zip](https://github.com/Universal-Team/LeafEdit/files/4259244/ACWW-Saves.zip)


Here are 3 Saves from 3 regions for testing too. EUR, USA and Japanese. I might end up doing Hex researches About that Issue.

**Comment by SuperSaiyajinStackZ** [comment]
The Issue seems to be fixed now on my end. The Name has overwritten the Gender offset. That still needs testing though.
